event-bus/src/kafka.rs

```diff
@@ -28,16 +28,23 @@ impl From<KafkaError> for Error {
 }
 
 impl KafkaEventBus {
-    pub(crate) fn new(brokers: String) -> Result<Self, Error> {
-        let producer: FutureProducer = ClientConfig::new()
-            .set("message.timeout.ms", "5000")
-            .set("bootstrap.servers", &brokers)
-            .create()?;
-        Ok(Self { brokers, producer })
+    pub(crate) fn new(brokers: String, properties: Vec<(String, String)>) -> Result<Self, Error> {
+        let mut config = ClientConfig::new();
+        config.set("bootstrap.servers", &brokers);
+
+        for (key, value) in properties {
+            config.set(key, value);
+        }
+
+        let mut producer_config = config.clone();
+        producer_config.set("message.timeout.ms", "5000");
```

Comment:
Why not pass pass this value and use 5000 as a default in case of missing value ? Otherwise this become a non configurable

Reply:
In this case (same for the other) we would need two additional config sections. (Common, Producer, Consumer). That wouldn't be wrong, but I think for the moment we should keep getting it working.